src/main/scala/io/getclump/Utils.scala

```diff
@@ -0,0 +1,13 @@
+package io.getclump
+
+private[getclump] case class RunOnce[T, U](f: T => Clump[U]) extends (T => Clump[U]) {
```

Comment:
This is necessary because of the seemingly innocuous modification of changing `val partial` to `def partial`. I've demonstrated the problem here: https://github.com/getclump/clump/pull/101

It needs to be a `def` though to allow the execution context to be passed in...